### ISSUE TITLE
Python library bug in error.py

### ISSUE THREAD

**Issue by monological** [elevenlabs/elevenlabs-python#73]
I'm using the python lib to make an API call, which occasionally throws an error, but is not being handled correctly in the library:

```
env/lib/python3.10/site-packages/elevenlabs/api/error.py", line 23, in __init__
    self.message = self.message or http_error.message
AttributeError: 'str' object has no attribute 'message
```

**Comment by flavioschneider** [comment]
Hi! Can you provide an example code that raises this error?

**Comment by monological** [comment]
I call this code many times and it usually works. Occasionally though this bug pops up instead of properly raising one of the valid exceptions: ElevenLabsHTTPError, ElevenLabsAPIError, or ElevenLabsRateLimitError
        
```
set_elevenlabs_api_key(elevenlabs_key)

        elevenlabs_voice = clone(
            name=uuid,
            description="",
            files=segment_file_paths,
            labels=None,
        )
```

**Comment by monological** [comment]
I forgot, it's also called here: 

```
set_elevenlabs_api_key(elevenlabs_key)
audio: bytes = generate(text, voice=elevenlabs_voice_id)
```

**Comment by flavioschneider** [comment]
This should be fixed in `v0.2.24`, can you check if it's working for you?
